metrics.go

```diff
@@ -6,32 +6,34 @@ import (
 )
 
 const (
-	ErrorRequestBodyCounter         = "error_request_body_count"
-	EmptyRequestBodyCounter         = "empty_request_body_count"
-	ModifiedWRPCounter              = "modified_wrp_count"
-	DeliveryCounter                 = "delivery_count"
-	DeliveryRetryCounter            = "delivery_retry_count"
-	DeliveryRetryMaxGauge           = "delivery_retry_max"
-	SlowConsumerDroppedMsgCounter   = "slow_consumer_dropped_message_count"
-	SlowConsumerCounter             = "slow_consumer_cut_off_count"
-	IncomingQueueDepth              = "incoming_queue_depth"
-	IncomingEventTypeCounter        = "incoming_event_type_count"
-	DropsDueToInvalidPayload        = "drops_due_to_invalid_payload"
-	OutgoingQueueDepth              = "outgoing_queue_depths"
-	DropsDueToPanic                 = "drops_due_to_panic"
-	ConsumerRenewalTimeGauge        = "consumer_renewal_time"
-	ConsumerDeliverUntilGauge       = "consumer_deliver_until"
-	ConsumerDropUntilGauge          = "consumer_drop_until"
-	ConsumerDeliveryWorkersGauge    = "consumer_delivery_workers"
-	ConsumerMaxDeliveryWorkersGauge = "consumer_delivery_workers_max"
-	QueryDurationSecondsHistogram   = "query_duration_seconds_histogram"
+	ErrorRequestBodyCounter               = "error_request_body_count"
+	EmptyRequestBodyCounter               = "empty_request_body_count"
+	ModifiedWRPCounter                    = "modified_wrp_count"
+	DeliveryCounter                       = "delivery_count"
+	DeliveryRetryCounter                  = "delivery_retry_count"
+	DeliveryRetryMaxGauge                 = "delivery_retry_max"
+	SlowConsumerDroppedMsgCounter         = "slow_consumer_dropped_message_count"
+	SlowConsumerCounter                   = "slow_consumer_cut_off_count"
+	IncomingQueueDepth                    = "incoming_queue_depth"
+	IncomingEventTypeCounter              = "incoming_event_type_count"
+	DropsDueToInvalidPayload              = "drops_due_to_invalid_payload"
+	OutgoingQueueDepth                    = "outgoing_queue_depths"
+	DropsDueToPanic                       = "drops_due_to_panic"
+	ConsumerRenewalTimeGauge              = "consumer_renewal_time"
+	ConsumerDeliverUntilGauge             = "consumer_deliver_until"
+	ConsumerDropUntilGauge                = "consumer_drop_until"
+	ConsumerDeliveryWorkersGauge          = "consumer_delivery_workers"
+	ConsumerMaxDeliveryWorkersGauge       = "consumer_delivery_workers_max"
+	QueryDurationSecondsHistogram         = "query_duration_seconds_histogram"
+	IncomingQueueLatencyHistogram_seconds = "incoming_queue_latency_histogram"
```

Comment:
did we mean to put `_seconds` in the actual value of `IncomingQueueLatencyHistogram_seconds`?